march_joint_inertia_controller/include/march_joint_inertia_controller/inertia_estimator.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 Project March.
+
+#ifndef MARCH_WS_INERTIA_ESTIMATOR_H
+#define MARCH_WS_INERTIA_ESTIMATOR_H
+
+#include <hardware_interface/joint_command_interface.h>
+#include <ros/ros.h>
+#include <trajectory_interface/quintic_spline_segment.h>
+#include <urdf/model.h>
+
+/**
+ * \brief determines the absolute value of vector a and stores it into b so that a remains unchanged
+ *
+ * @param[in] a input vector with values that need to be converted to absolutes
+ * @param[out] b output vector with copies of the absolute values of a.
+ */
+void absolute(std::vector<double> a, std::vector<double>& b);
+
+/**
+ * \brief determiness the mean value of the given vector
+ *
+ * @param[in] a input vector from which the mean gets calculated
+ * @returns the mean value of a
+ */
+double mean(std::vector<double> a);
```

Comment:
Same here, pass the vector as a const reference
```suggestion
double mean(const std::vector<double>& a);
```